requirements.txt

```diff
@@ -0,0 +1,9 @@
+bcrypt==4.0.1
+Flask==2.2.3
+Flask-session2==1.3.1
+Flask-SQLAlchemy==3.0.3
+Flask-wtf==1.1.1
+psycopg2-binary==2.9.5
+pytest-flask==1.2.0
```

Comment:
We could move the test requirements to setup.py (https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies ):

```python
extras_require={
        "test": ["pytest-flask==1.2.0", "pytest==7.2.1"],
},
```

Installing the requirements:
`python -m pip install .[test]`

Then we do not have to manage the requirements in setup.py and requirements.txt.